test/e2e/framework/skipper/skipper.go

```diff
@@ -224,10 +224,10 @@ func SkipIfMultizone(c clientset.Interface) {
 	}
 }
 
-// SkipUnlessMasterOSDistroIs skips if the master OS distro is not included in the supportedMasterOsDistros.
-func SkipUnlessMasterOSDistroIs(supportedMasterOsDistros ...string) {
-	if !framework.MasterOSDistroIs(supportedMasterOsDistros...) {
-		skipInternalf(1, "Only supported for master OS distro %v (not %s)", supportedMasterOsDistros, framework.TestContext.MasterOSDistro)
+// SkipUnlessControlPlaneOSDistroIs skips if the controlplane OS distro is not included in the supportedControlPlaneOsDistros.
+func SkipUnlessControlPlaneOSDistroIs(supportedControlPlaneOsDistros ...string) {
+	if !framework.ControlPlaneOSDistroIs(supportedControlPlaneOsDistros...) {
+		skipInternalf(1, "Only supported for controlplane OS distro %v (not %s)", supportedControlPlaneOsDistros, framework.TestContext.ControlPlaneOSDistro)
```

Comment:
I found that you have renamed SkipUnlessMasterOSDistroIs to SkipUnlessControlPlaneOSDistroIs. But I can not find where is calling SkipUnlessControlPlaneOSDistroIs.